docs/concepts/assets/accounts.md

```diff
@@ -24,6 +24,39 @@ Any assets that are in a general account can be withdrawn or transferred.
 **[REST](../../api/rest/data-v2/trading-data-service-get-party.api.mdx)**: Use the API to check your public key for the accounts your assets are allocated to.
 :::
 
+## Vault
```

Comment:
what about documenting the vault management operations, create/update/change ownership?